src/Engine/Engine.cpp

```diff
@@ -767,6 +767,8 @@ void Engine::SecondaryInitialization() {
     pBitmaps_LOD->reserveLoadedTextures();
     pSprites_LOD->reserveLoadedSprites();
 
+    pActors.reserve(engine->config->gameplay.MaxActors.value());
```

Comment:
pActors was growing whilst iterating through invalidating pointers. Better place to put this or better fix?